### ISSUE TITLE
Patch DevModeInitializer upstream

### ISSUE THREAD

**Issue by mvysny** [mvysny/vaadin-quarkus#1]
The `DevModeInitializer.java` (which comes from Vaadin Flow) is not compatible with Quarkus ClassLoader hierarchy.

The code fails with ClassNotFoundException in the `isVaadinServletSubClass()` method since `MyServlet` is loaded via a child classloader.

The function needs to be patched to use the context class loader:

```java
    private boolean isVaadinServletSubClass(String className)
            throws ClassNotFoundException {
        return VaadinServlet.class.isAssignableFrom(Class.forName(className, true, Thread.currentThread().getContextClassLoader()));
    }
```

**Comment by mvysny** [comment]
Reported upstream as https://github.com/vaadin/flow/issues/9755

**Comment by mvysny** [comment]
Closed: the custom Servlet was used to help Vaadin locate the `flow-build-info.json` (a workaround for #2 ), however another workaround was found. Closed as duplicate of #2.
